sources/packages/backend/apps/queue-consumers/src/processors/assessment/start-application-assessment.processor.ts

```diff
@@ -31,6 +32,22 @@ export class StartApplicationAssessmentProcessor {
     const processSummary = new ProcessSummary();
     try {
       processSummary.info("Processing the start assessment job.");
+      const assessment = await this.studentAssessmentService.getAssessmentById(
+        job.data.assessmentId,
+      );
+      if (
+        assessment.studentAssessmentStatus !== StudentAssessmentStatus.Queued
+      ) {
+        await job.discard();
+        processSummary.warn(
+          `Assessment id ${job.data.assessmentId} is not in ${StudentAssessmentStatus.Queued} status.`,
+        );
+        const endProcessMessage =
+          "Workflow process not executed due to the assessment not being in the correct status.";
+        processSummary.info(endProcessMessage);
```

Comment:
why is this info and line 42 warn?

Reply:
This method is returning `summary.getSummary() `. If this endProcessMessage is not info it's not included in the summary.